modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -402,6 +403,12 @@ public void testGetObjectEntries() throws Exception {
 			).build(),
 			objectEntry2, objectEntry3, objectEntry4);
 
+		testGetObjectEntries(
+			HashMapBuilder.put(
+				"filter", "title eq null"
```

Comment:
Hey @jairmedeiros use `buildEqualsExpressionFilterString` here, I believe you have to adapt the method [getValue](https://github.com/liferay/liferay-portal/blob/08475a044218ee312bc6452f6348f6d686cf8f6c/modules/apps/object/object-rest-test-util/src/main/java/com/liferay/object/rest/test/util/BaseObjectEntryManagerImplTestCase.java#L118) to handle the null use case.

Thank you"